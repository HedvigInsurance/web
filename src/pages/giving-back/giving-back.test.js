import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

import GivingBack from './index';

describe('<GivingBack />', () => {
  it('Should render correctly', () => {
    const component = shallow(<GivingBack />);

    expect(toJson(component)).toMatchSnapshot();
  });
});
