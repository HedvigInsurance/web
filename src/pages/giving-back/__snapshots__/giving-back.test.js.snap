// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<GivingBack /> Should render correctly 1`] = `
<main
  className="Site"
>
  <Header
    isInverted={false}
  />
  <article
    className="Site-content"
  >
    <div
      className="u-backgroundSecondaryPurple"
    >
      <div
        className="Container"
      >
        <h1
          className="u-spaceMT2 u-spaceMB8 u-md-spaceMB6 u-lg-spaceMB6 u-fontFamilyHeader u-fontSize5 u-md-fontSize3 u-lg-fontSize2"
        >
          Hur vi ger tillbaka
        </h1>
      </div>
    </div>
    <div
      className="Container u-md-spaceMT10 u-lg-spaceMT10 u-spaceMB5 u-md-spaceMB3 u-lg-spaceMB3"
    >
      <div
        className="u-maxWidth1of1"
      >
        <h2
          className="u-spaceMT5 u-spaceMB10 u-fontFamilyHeader u-fontSize8 u-md-fontSize7 u-lg-fontSize7"
        >
          Vårt sätt att bidra
        </h2>
        <p
          className="u-lg-fontSize9 u-spaceMB9"
        >
          Försäkring finns till för att göra livet lättare för människor. Vår prio ett är alltid våra medlemmar, men blir det pengar över kan vi tillsammans vara med och bidra.
        </p>
        <h2
          className="u-spaceMT5 u-spaceMB10 u-fontFamilyHeader u-fontSize8 u-md-fontSize7 u-lg-fontSize7"
        >
          En schysstare modell för alla
        </h2>
        <p
          className="u-spaceMB9 u-lg-fontSize9"
        >
          Hedvig tar en fast avgift oavsett hur mycket ersättning som betalas ut. Överskottet skänks till ett gott ändamål istället för att gå till extra vinst.
        </p>
        <h3
          className="u-spaceMT8 u-spaceMB10 u-fontSize11 u-md-fontSize10 u-lg-fontSize9"
        >
          Detta betyder att:
        </h3>
        <ul
          className="u-spaceMT10"
        >
          <li
            className="u-fontSize9 u-spaceMB10"
            style={
              Object {
                "paddingLeft": "2em",
                "textIndent": "-2em",
              }
            }
          >
            <ReactComponent
              className="u-inlineBlock u-spaceMR11"
              style={
                Object {
                  "marginRight": 10,
                  "verticalAlign": "text-bottom",
                  "width": 20,
                }
              }
            />
            Du bidrar till att skapa en bättre värld
          </li>
          <li
            className="u-fontSize9 u-spaceMB10"
            style={
              Object {
                "paddingLeft": "2em",
                "textIndent": "-2em",
              }
            }
          >
            <ReactComponent
              className="u-inlineBlock u-spaceMR11"
              style={
                Object {
                  "marginRight": 10,
                  "verticalAlign": "text-bottom",
                  "width": 20,
                }
              }
            />
            Du kan känna dig trygg att du får rätt ersättning, eftersom Hedvig inte tjänar på att betala ut mindre till dig
          </li>
        </ul>
      </div>
      <div>
        <h2
          className="u-spaceMT5 u-spaceMB10 u-fontFamilyHeader u-fontSize8 u-md-fontSize7 u-lg-fontSize7"
        >
          Så funkar det
        </h2>
        <div
          className="Grid Grid--withGutter"
        >
          <div
            className="u-md-size1of3 u-lg-size1of3"
          >
            <span
              className="NumberedIcon"
            >
              1
            </span>
            <p
              className="u-lg-fontSize9 u-spaceMB9"
            >
              Skaffa Hedvig och välj det ändamål du vill stötta
            </p>
          </div>
          <div
            className="u-md-size1of3 u-lg-size1of3"
          >
            <span
              className="NumberedIcon"
            >
              2
            </span>
            <p
              className="u-lg-fontSize9 u-spaceMB9"
            >
              Vid årets slut summerar vi alla pengar som inte betalats ut i ersättningar till dig, eller till andra som valt samma ändamål
            </p>
          </div>
          <div
            className="u-md-size1of3 u-lg-size1of3"
          >
            <span
              className="NumberedIcon"
            >
              3
            </span>
            <p
              className="u-lg-fontSize9 u-spaceMB9"
            >
              Tillsammans gör vi skillnad genom att skänka pengarna
            </p>
          </div>
        </div>
      </div>
      <div>
        <h2
          className="u-spaceMT5 u-spaceMB10 u-fontFamilyHeader u-fontSize8 u-md-fontSize7 u-lg-fontSize7"
        >
          Våra partners
        </h2>
        <div
          className="Grid Grid--withGutter"
        >
          <div
            className="u-spaceMB8 u-md-size1of2 u-lg-size1of2 u-flex u-flexCol u-flexAlignItemsCenter"
          >
            <figure
              className="u-flex u-flexCol u-flexJustifyCenter u-spaceMB7"
              style={
                Object {
                  "height": 120,
                }
              }
            >
              <ReactComponent />
            </figure>
            <p
              className="u-lg-fontSize9 u-spaceMB9 u-textCenter u-maxWidth1of2"
            >
              SOS Barnbyar arbetar för att fler barn ska få en trygg uppväxt. Stödet från våra medlemmar går till att finansiera en barnby i Ukraina.
            </p>
          </div>
          <div
            className="u-spaceMB8 u-md-size1of2 u-lg-size1of2 u-flex u-flexCol u-flexAlignItemsCenter"
          >
            <figure
              className="u-spaceMB7"
              style={
                Object {
                  "height": 120,
                }
              }
            >
              <ReactComponent />
            </figure>
            <p
              className="u-lg-fontSize9 u-spaceMB9 u-textCenter u-maxWidth1of2"
            >
              Barncancerfonden arbetar för att bekämpa cancer och för att cancerdrabbade barn, ungdomar och deras familjer ska få den vård och det stöd de behöver.
            </p>
          </div>
        </div>
      </div>
      <div>
        <div
          className="Grid Grid--withGutter Grid--alignCenter u-spaceMT8"
        >
          <a
            className="Button u-colorWhite u-lg-fontSize9 u-backgroundPrimaryGreen"
            data-cta-app-download="giving-back"
            href="https://hedvig.app.link/kZNtW0cT9L"
          >
            Skaffa Hedvig
          </a>
        </div>
      </div>
    </div>
  </article>
  <Footer />
</main>
`;
